### DISCUSSION TITLE
Do you want Code to open the external website?

### DISCUSSION THREAD

**Discussion by jamescampagnola** [community/community#58344] [Codespaces]
### Select Topic Area

General

### Body

I'm working in a codespace and I continually get a message window:  "Do you want Code to open the external website?"  I don't have a clue what this is. What is this and what does it mean?  Thank you very much for your assistance.  JIm

**Comment by tiwament1477** [comment]
Yes i want that

**Comment by dmgardiner25** [comment]
@jamescampagnola Can you share what URL is trying to be opened? It should show in the modal that pops up.

**Comment by jamescampagnola** [comment]
After loading my Codespace, a window periodically appears at the bottom of the screen in the terminal window:  'Please visit https://cs50.dev and log in again.’ There is one button- ‘OK’.  Pressing this button opens another window which says:  ‘Do you want to open the external website? https://cs50.dev/' <https://cs50.dev/'>.  In this window, there are 4 buttons to choose from-    ‘Configure Trusted Domains,’  ‘Copy,’ ‘Cancel,’ and ‘Open’.  If I select ‘Open’, I am taken out of the Codespace to a URL: cs50.dev, which is “CS50’s adaptation of Codespaces for students and teachers.” There is a Log In button which features a dropdown menu with the choices- ‘ Log In’ (default) - ‘Open in VS Code Desktop’ - ‘Restart Codespace’ - ‘Backups’ - ‘Commits’ - and  ‘Repository.’  If I press ‘Log In,’ My Codespace reloads and another window appears at the bottom of the screen again in the terminal section:  “Updates Available.” Source: CS50 (Extension).  I click “Update Now” and nothing seems to happen.





After a short period of time, another window opens in the terminal section of the screen: "Remote repository contains conflicting changes.  Source: GitDoc (Extension)”. There is a button- “Force Push.”  There is also a ‘gear’ icon which says- “Manage Extension.”  When I press this button, nothing happens and a few seconds later the same window appears again.  This time I X out of the window and nothing happens.

The above routine happens constantly while I’m working, wastes a lot of time and is very distracting.  Can anyone help with this or at least explain what’s happening.  Thanks.

Jim
